pom.xml

```diff
@@ -109,6 +110,27 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>1.9.59</version>
       </plugin>
+      
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+          </execution>
+        </executions>
+
+        <configuration>
+          <executable>npm</executable>
+          <arguments>
+            <argument>install</argument>
```

Comment:
This now requires our devs to have npm install globally. Is there a maven plugin that manages this, or could we at least add this to our readme?